composer.json

```diff
@@ -29,30 +29,10 @@
     "repositories": [
         {
             "type": "vcs",
-            "url": "https://github.com/wallabag/PHP-Flash-Messages"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/wallabag/kriss_php5"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/wallabag/pagination"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/wallabag/PHPePub"
-        },
-        {
-            "type": "vcs",
             "url": "https://github.com/wallabag/php-readability"
```

Comment:
Oh sorry I was thinking that removing all lines and just add tag to require will be enough, my bad.